scRNA-seq/04-tag-based_scRNA-seq_processing.Rmd

```diff
@@ -0,0 +1,246 @@
+10X_P4_3_L001_R2_001.fastq.gz  10X_P4_3_L002_R2_001.fastq.gz
+10X_P4_3_L001_R2_002.fastq.gz  10X_P4_3_L002_R2_002.fastq.gz
+10X_P4_3_L001_R2_003.fastq.gz  10X_P4_3_L002_R2_003.fastq.gz
+```
+
+We should really run all of these through Salmon, though that will take ~six times as long as the single pair of reads we used. 
+To do this, we could list each R1 and R2 file (space separated) after the `-1` and `-2` arguments, respectively.
+But that is a lot of typing, so a nice shortcut is to use a `*` character to represent a wildcard that will be filled in with whatever characters are present in the files at the given path.
+In the pattern `10X_P4_3_L*_R1_*.fastq.gz`, this would allow any lane number and any subset, so would match all of the following files (all the R1 files in this case):
+
+```
+10X_P4_3_L001_R1_001.fastq.gz  10X_P4_3_L002_R1_001.fastq.gz
+10X_P4_3_L001_R1_002.fastq.gz  10X_P4_3_L002_R1_002.fastq.gz
+10X_P4_3_L001_R1_003.fastq.gz  10X_P4_3_L002_R1_003.fastq.gz
+```
+
+For this directory, that would look make our full `salmon alevin` command look like this (don't run this now!):
+
+```
+salmon alevin -l ISR \
+  -i /shared/data/reference/refgenie/mm10_cdna/salmon_index/short/short/ \
+  -1 data/tabula-muris/fastq-raw/10X_P4_3/10X_P4_3_L*_R1_*.fastq.gz \
+  -2 data/tabula-muris/fastq-raw/10X_P4_3/10X_P4_3_L*_R2_*.fastq.gz \
+  --chromium  \
+  -p 4 \
+  -o data/tabula-muris/alevin-quant/10X_P4_3_full \
+  --tgMap data/tabula-muris/mm_ensdb95_tx2gene.tsv \
+  --dumpFeatures
+```
+
+In general, you will want to run all lanes and all files for a given sample together.
+But DO NOT want to combine multiple samples into a single `alevin` quantification!
+
+## Step 6: Perform QC checks with `alevinQC`
+
+Now that we have quantified our data with Alevin, we are ready to perform
+quality control checks.
+
+![**Current status of this dataset**](diagrams/tag-based_2.png)
+
+In order to perform quality control checks, we'll use the `alevinQC` R package.
+Note that `alevinQC` depends on files that we get using the`--dumpFeatures`
+option in Alevin.
+
+About the `alevinQCReport` function:
+The first argument needs to be where the sample's output data was put when
+Alevin was run (as a character string, aka using quotes)
+The rest of `alevinQCReport`'s arguments tell R where to put the output QC
+report also using `characters`.
+
+```{r}
+# Produce a QC report
+alevinQC::alevinQCReport("data/tabula-muris/alevin-quant/10X_P4_3",
+                         sampleId = "10X_P4_3",
+                         outputFile = "10X_P4_3_qc_report.html",
+                         outputDir = "qc-reports")
+```
+
+Check out `data/tabula_muris/qc_reports/10X_P4_3_subset_qc_report.html`
+in order to examine the quality of your data and performance of Alevin.
+
+
+This is an example of [a poor quality sample alevinQC report](https://alexslemonade.github.io/training-modules/scRNA-seq/data/tabula_muris/qc_reports/Bad_Example_10X_P4_2_qc_report.html).
```

Comment:
If we still want to use this link, I would expect `scRNA-seq/data/tabula_muris/qc_reports/Bad_Example_10X_P4_2_qc_report.html` to be updated in this pull request.